internal/policies/mount/internal_test.go

```diff
@@ -0,0 +1,70 @@
+package mount
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var update bool
+
+func TestWriteMountsFile(t *testing.T) {
+	t.Parallel()
+
+	tests := map[string]struct {
+		path        string
+		nEntries    int
+		nErrored    int
+		nValues     int
+		nDuplicated int
+		separators  []string
+
+		wantErr bool
+	}{
+		"write mounts file with multiple entries":                          {nEntries: 5, nValues: 1},
+		"write mounts file with values separated by ','":                   {nEntries: 1, nValues: 5, separators: []string{","}},
```

Comment:
I would add some tests with one entry and one value. Also, maybe one entry, no value.